src/test/kotlin/lotto/domain/LottoStoreTest.kt

```diff
@@ -18,4 +18,36 @@ internal class LottoStoreTest {
         // then
         assertThat(lottos.size).isEqualTo(expected)
     }
+
+    @Test
+    @DisplayName("자동, 수동을 섞어서 로또를 구매할 수 있다")
+    fun autoAndManualBuyTest() {
+        // given
+        val money = Money(14000)
+        val manualLottos = listOf(
+            Lotto.of(setOf(1, 2, 3, 4, 5, 6)),
+            Lotto.of(setOf(1, 2, 3, 4, 5, 6)),
+            Lotto.of(setOf(1, 2, 3, 4, 5, 6))
+        )
+        // when
+        val lottos = LottoStore().buy(money, manualLottos)
+        // then
+        assertThat(lottos).containsAnyElementsOf(manualLottos)
+    }
+
+    @Test
+    @DisplayName("수동으로 로또를 구매할 수 있다")
+    fun manualBuyTest() {
+        // given
+        val money = Money(3000)
+        val manualLottos = listOf(
+            Lotto.of(setOf(1, 2, 3, 4, 5, 6)),
+            Lotto.of(setOf(1, 2, 3, 4, 5, 6)),
+            Lotto.of(setOf(1, 2, 3, 4, 5, 6))
+        )
+        // when
+        val lottos = LottoStore().buy(money, manualLottos)
+        // then
+        assertThat(lottos).containsExactlyElementsOf(manualLottos)
```

Comment:
수동 로또만 구매하는 상황에 대해 테스트를 작성해주신 것 같네요.
자동 로또는 포함되면 안되는 상황이니 `containsExactlyElementsOf`가 아닌 isEqualTo로 자동 로또만 포함되었는지 검증하면 어떨까요?

```suggestion
        assertThat(lottos).isEqualTo(manualLottos)
```